registry/xml/al.xml

```diff
@@ -0,0 +1,5469 @@
+                <command name="alGenFilters"/>
+                <command name="alDeleteFilters"/>
+                <command name="alIsFilter"/>
+                <command name="alFilteri"/>
+                <command name="alFilteriv"/>
+                <command name="alFilterf"/>
+                <command name="alFilterfv"/>
+                <command name="alGetFilteri"/>
+                <command name="alGetFilteriv"/>
+                <command name="alGetFilterf"/>
+                <command name="alGetFilterfv"/>
+            </require>
+            <require api="al" comment="Auxiliary Effect Slot object function types.">
+                <command name="alGenAuxiliaryEffectSlots"/>
+                <command name="alDeleteAuxiliaryEffectSlots"/>
+                <command name="alIsAuxiliaryEffectSlot"/>
+                <command name="alAuxiliaryEffectSloti"/>
+                <command name="alAuxiliaryEffectSlotiv"/>
+                <command name="alAuxiliaryEffectSlotf"/>
+                <command name="alAuxiliaryEffectSlotfv"/>
+                <command name="alGetAuxiliaryEffectSloti"/>
+                <command name="alGetAuxiliaryEffectSlotiv"/>
+                <command name="alGetAuxiliaryEffectSlotf"/>
+                <command name="alGetAuxiliaryEffectSlotfv"/>
+            </require>
+            <require api="al" comment="Lowpass filter">
+                <enum name="AL_LOWPASS_MIN_GAIN"/>
+                <enum name="AL_LOWPASS_MAX_GAIN"/>
+                <enum name="AL_LOWPASS_DEFAULT_GAIN"/>
+            </require>
+            <require api="al">
+                <enum name="AL_LOWPASS_MIN_GAINHF"/>
+                <enum name="AL_LOWPASS_MAX_GAINHF"/>
+                <enum name="AL_LOWPASS_DEFAULT_GAINHF"/>
+            </require>
+            <require api="al" comment="Highpass filter">
+                <enum name="AL_HIGHPASS_MIN_GAIN"/>
+                <enum name="AL_HIGHPASS_MAX_GAIN"/>
+                <enum name="AL_HIGHPASS_DEFAULT_GAIN"/>
+            </require>
+            <require api="al">
+                <enum name="AL_HIGHPASS_MIN_GAINLF"/>
+                <enum name="AL_HIGHPASS_MAX_GAINLF"/>
+                <enum name="AL_HIGHPASS_DEFAULT_GAINLF"/>
+            </require>
+            <require api="al" comment="Bandpass filter">
+                <enum name="AL_BANDPASS_MIN_GAIN"/>
+                <enum name="AL_BANDPASS_MAX_GAIN"/>
+                <enum name="AL_BANDPASS_DEFAULT_GAIN"/>
+            </require>
+            <require api="al">
+                <enum name="AL_BANDPASS_MIN_GAINHF"/>
+                <enum name="AL_BANDPASS_MAX_GAINHF"/>
+                <enum name="AL_BANDPASS_DEFAULT_GAINHF"/>
+            </require>
+            <require api="al">
+                <enum name="AL_BANDPASS_MIN_GAINLF"/>
+                <enum name="AL_BANDPASS_MAX_GAINLF"/>
+                <enum name="AL_BANDPASS_DEFAULT_GAINLF"/>
+            </require>
+            <r
```

Comment:
Typo, should be `AL_SOFT_buffer_samples`.

Reply:
Hmm, not sure what's happening with the line references. It should be 4815, not 4241...4244.